Yosemite/YosemiteTests/Stores/NotificationCountStoreTests.swift

```diff
@@ -0,0 +1,166 @@
+import XCTest
+
+@testable import Storage
```

Comment:
This import does not seem to be needed.

```suggestion

```

Reply:
yup updated in https://github.com/woocommerce/woocommerce-ios/pull/2320/commits/674b6fbf3cb42be622dfaad802ab34e20d3b3b7c